app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class ZIPCode(db.Model):
+    __tablename__ = 'ZIPCodes'
+    id = db.Column(db.Integer, primary_key=True)
+    zipcode = db.Column(db.Integer, unique=True)
+    users = db.relationship('ZIPCode', backref='user', lazy='dynamic')
+    resources = db.relationship('ZIPCode', backref='resource', lazy='dynamic')
+
+    def __repr__(self):
+        return '<ZIPCode \' %s \'>' % self.zipcode
+
+
+class Address(db.Model):
+    __tablename__ = 'addresses'
+    id = db.Column(db.Integer, primary_key=True)
+    # http://pe.usps.com/text/pub28/28c2_001.htm
```

Comment:
Preface this comment with context? Something like "USPS Addressing Standards" perhaps.